velox/docs/functions/presto/math.rst

```diff
@@ -238,6 +238,10 @@ Floating Point Functions
 
     Returns the constant representing not-a-number.
 
+.. function:: weibull_cdf(a, b, value) -> double
```

Comment:
This function should be in the next section with Probability functions.

Reply:
Fixed.